dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/administrative/collection/EditCollectionMetadataForm.java

```diff
@@ -209,8 +210,8 @@ public void addBody(Body body)
 	    
 		Para buttonList = main.addPara();
 	    buttonList.addButton("submit_save").setValue(T_submit_save);
-        //Only System Admins can Delete Collections
-	    if (authorizeService.isAdmin(context))
+
+	    if (AuthorizeManager.authorizeActionBoolean(context, thisCollection, Constants.DELETE))
```

Comment:
Hi @dimitrove , thanks for creating your first Pull Request (PR)!

One note here, it looks like Travis CI noticed an error in the code (hence the red x above).  In the DSpace 6 codebase, there's no longer a `AuthorizeManager` class.  It's been replaced by an `AuthorizeService`.

So, in this line of the code, I think you need it to read:
`authorizeService.authorizeActionBoolean(context, thisCollection, Constants.DELETE)`

Beyond that, it looks like you did everything right with this PR!  You should be able to simply add a new commit to your "DS-3254" fixing that line of code, and this PR should update automatically.